### PR TITLE
Lm judge bug fix

### PR DESCRIPTION
**Pull request by mvcrouse**
## What this PR does / why we need it

Fixes lm_judge class to update with block design

## If applicable**
- [ ] this PR contains documentation
- [X] this PR contains unit tests
- [X] this PR has been tested for backwards compatibility

### CODE REVIEW

File: fms_dgt/blocks/validators/lm_judge.py [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+
+        for llm_output in llm_outputs:
+            lm_res = self.get_result(llm_output, result_field)
+            new_result = success_func(lm_res)
+            self.write_result(llm_output, new_result, result_field=result_field)
+
+        return llm_outputs
+
+    def _validate(self, lm_output: str, success_func: Callable) -> bool:
+        return success_func(lm_output)
```
Comment (sivasankalpp) [inline fms_dgt/blocks/validators/lm_judge.py:55]: I see you've removed `validate_batch`. Is the intention to replace these calls with `generate`?

Comment (mvcrouse) [reply fms_dgt/blocks/validators/lm_judge.py:55]: Yeah, all blocks will have the `generate` call as their main entry point. Which, hopefully that will change at some point, but want to be compatible with IL

**Review by sivasankalpp — APPROVED** [review]
LGTM

File: fms_dgt/blocks/validators/lm_judge.py (hunk 1 above)
Comment (sivasankalpp) [reply fms_dgt/blocks/validators/lm_judge.py:55]: Fair enough

**Pull request merged**